### PR TITLE
commonlib: migrate timing utils from ascanrules

### PR DESCRIPTION
**Pull request by itsdivyanshjain**
## Overview
Briefly describe the purpose, goals, and changes or improvements made in this pull request.

## Related Issues
Specify any related issues or pull requests by linking to them.

## Checklist
- [NA] Update help
- [X] Update changelog
- [X] Run `./gradlew spotlessApply` for code formatting
- [NA] Write tests
- [NA] Check code coverage
- [x] Sign-off commits
- [ ] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

**Comment by itsdivyanshjain** [comment]
@thc202 @kingthorin do review this

File: addOns/ascanrules/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Changed
+- Depend on newer version of Common Library add-on.
+- Re-work Command Injection to handle timing utils changes.
```
Comment (thc202) [inline addOns/ascanrules/CHANGELOG.md:9]: Remove this entry.

Comment (thc202) [reply addOns/ascanrules/CHANGELOG.md:9]: Or if you want to be explicit about the fix, mention the scan rule in the fixed entry.

Comment (itsdivyanshjain) [reply addOns/ascanrules/CHANGELOG.md]: Done

**Comment by kingthorin** [comment]
The PR seems to be more than the title states.

**Comment by thc202** [comment]
@kingthorin it looks fine, just moving the classes around.

**Comment by kingthorin** [comment]
When I looked earlier I thought there were still more Command Injection changes, sorry.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
Thanks!

**Comment by thc202** [comment]
@itsdivyanshjain how would you like to be credited (e.g. name, handle)?
https://www.zaproxy.org/docs/desktop/credits/#zap-extended-team

**Comment by itsdivyanshjain** [comment]
> @itsdivyanshjain how would you like to be credited (e.g. name, handle)? https://www.zaproxy.org/docs/desktop/credits/#zap-extended-team

Divyansh Jain ([@itsdivyanshjain](https://twitter.com/itsdivyanshjain) ) - Astra Security

**Pull request merged**